<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RWScanRect</class>
 <widget class="QMainWindow" name="RWScanRect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RWScanRect</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="imageTitleL">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Bilder</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="imageAddPB">
            <property name="text">
             <string>Hinzufügen...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="imageLV">
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::DragDrop</enum>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="imagePropertyGB">
          <property name="title">
           <string>Bildeigenschaften</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="imagePropertyL">
             <property name="text">
              <string>(Text)</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="documentTitleL">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Dokumente</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="documentAddPB">
            <property name="text">
             <string>Neu</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeView" name="documentTV">
          <property name="dragDropMode">
           <enum>QAbstractItemView::DragDrop</enum>
          </property>
          <property name="defaultDropAction">
           <enum>Qt::TargetMoveAction</enum>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="documentPropertyGB">
          <property name="title">
           <string>Dokumenteigenschaften</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="documentPropertyL">
             <property name="text">
              <string>(Text)</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTabWidget" name="viewTW">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="rawImageTAB">
        <attribute name="title">
         <string>Rohbild</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="leftMargin">
            <number>9</number>
           </property>
           <property name="topMargin">
            <number>9</number>
           </property>
           <property name="rightMargin">
            <number>9</number>
           </property>
           <property name="bottomMargin">
            <number>9</number>
           </property>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Skalierung:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="rawImageScaleCB"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>360</width>
              <height>513</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="rawImageL">
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="processedImageTAB">
        <attribute name="title">
         <string>Korrigiertes Bild</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="documentTAB">
        <attribute name="title">
         <string>Dokument</string>
        </attribute>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>929</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileM">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="projectNewACT"/>
    <addaction name="projectOpenACT"/>
    <addaction name="projectSaveACT"/>
    <addaction name="projectSaveAsACT"/>
    <addaction name="separator"/>
    <addaction name="quitACT"/>
   </widget>
   <addaction name="fileM"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="projectOpenACT">
   <property name="text">
    <string>Projekt öffnen...</string>
   </property>
  </action>
  <action name="projectSaveACT">
   <property name="text">
    <string>Projekt speichern</string>
   </property>
  </action>
  <action name="projectSaveAsACT">
   <property name="text">
    <string>Projekt speichern unter...</string>
   </property>
  </action>
  <action name="quitACT">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
  <action name="projectNewACT">
   <property name="text">
    <string>Neues Projekt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
